governance.md

```diff
@@ -0,0 +1,393 @@
+quality and quantity
+-  Sustain this development consistently over several months
+-  Support the community through various activities (some examples below):
+   - code review
+   - answering user questions
+   - triaging bug reports
+   - participating constructively in broader conversations
+   - contributing to documentation
+   - maintaining infrastructure
+-  Demonstrate breadth by supporting the community outside of their particular
+   work interests or sub-project
+-  Be civil in public discourse
+
+Potential Council Members are nominated by existing Council members and voted
+upon by the existing Council after asking if the potential Member is interested
+and willing to serve in that capacity. The Council will be initially formed
+from the set of existing Core Developers who, as of 2019, have ownership rights
+over the organization.
+
+If a Council member becomes inactive in the project for a period of one year,
+they will be considered for removal from the Council. Before removal, the
+inactive Member will be approached to see if they plan to return
+to active participation. If not, they will be removed immediately upon a Council
+vote. If they plan to return to active participation soon, they will be
+given a grace period of one year. If they don’t return to active participation
+within that time period they will be removed by vote of the Council without
+further grace period. All former Council members can be considered for
+membership again at any time in the future, like any other Project Contributor.
+Retired Council members will be listed on the project website, acknowledging
+the period during which they were active in the Council.
+An inactive BDFL may also be ejected by a consensus of current members under
+these same conditions.
+
+The Council reserves the right to eject current Members, other than the BDFL,
+if they are deemed to be actively harmful to the project’s well-being, and
+attempts at communication and conflict resolution have failed. This action
+should not be taken lightly. The proposer must provide overwhelming
+justification as to why this action is needed (e.g. the Member violated the
+Code of Conduct) and not other technical reasons (e.g. we have different
+viewpoints on project direction) or personal reasons (e.g. they belong to a
+competitor's organization). We ask that a majority of owners respond for
+quorum, and that those votes come from people employed/associated with at least
+three institutions.
+
+
+### Conflict of interest
+
+It is expected that the Council Members and BDFL will be employed at a wide
+range of companies, universities and non-profit organizations. Because of this,
+it is possible that Members will have conflict of interests. Such conflict of
+interests include, but are not limited to:
+
+-   Financial interests, such as investments, employment, or contracting work,
+    outside of The Project that may influence their work on The Project.
```

Comment:
How is "investments, employment, or contracting work, outside of The Project" defined in this context?  An example might be helpful.

Reply:
I think this is referring to people's day jobs. Though I may not fully understand the question.

Reply:
I think that in general yes, this most common case here is people's day jobs.  I think it's ok to leave this vague and subject to interpretation by the coucil though.